diff --git a/configure.ac b/configure.ac
index c3f0b5b..a7129eb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1184,6 +1184,10 @@ AC_SUBST(GETTEXT_PACKAGE)
 ALL_LINGUAS="da de es et fi fr hu it ja ko nb nl pl pt pt_BR ru sk sv vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 MKINSTALLDIRS="$INSTALL_DIRS"
+dnl The po/Makefile.in.in created by `intltoolize` references @install_sh@,
+dnl which is subsituted by AM_PROG_INSTALL_SH at least... This fixes
+dnl installation of .mo files.
+AM_PROG_INSTALL_SH
 
 
 ###############################################################################
